tensorflow/compiler/mlir/tensorflow/ir/tf_saved_model.cc

```diff
@@ -400,6 +400,7 @@ LogicalResult VerifyExportedFunc(func::FuncOp func) {
       return func.emitError() << "'tf.resource_name' attribute is not allowed "
                                  "unless it is being under construction";
     }
+    if (func.getArgAttr(i, "emitc.hint_name")) continue;
```

Comment:
Where is this called from? E.g., if you remove this what's the stack trace reporting an error.